app/controllers/concerns/idv_step_concern.rb

```diff
@@ -60,7 +60,7 @@ def confirm_document_capture_complete
   def confirm_verify_info_step_complete
     return if idv_session.verify_info_step_complete?
 
-    if idv_session.in_person_enrollment?
```

Comment:
Here we will not have created the profile yet.

This is intended to send the user to the correct verify info step. That comes before we actually start writing records to user or profile record. We will have written to the proofing components which is presumably why we check for that here. I think that we are better leaving this portion alone for the profile state work.

There is perhaps some refactoring that could be done to mark a session as in-person proofing directly instead of looking for artifacts in the db. That doesn't really seem to fit into the scope here though.